cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -22,30 +22,30 @@
     <div class="m-global-banner">
         <div class="wrapper
                     wrapper__match-content
-                    m-expandable
-                    m-expandable__expanded">
+                    o-expandable
+                    o-expandable__expanded">
             <div class="m-global-banner_head">
                 <span class="cf-icon
                              cf-icon-error-round
                              m-global-banner_icon"></span>
                 This beta site is a work in progress.
             </div>
-            <div class="m-expandable_content">
-                <p class="m-global-banner_desc  m-expandable_content-animated">
+            <div class="o-expandable_content">
+                <p class="m-global-banner_desc  o-expandable_content-animated">
```

Comment:
Update with `o-expandable_content-animated` as a wrapper around `m-global-banner_desc`. Mixing two atomic components, especially two different types is a terrible lesson we already learned elsewhere.